tools/release_pull_request_template.md

```diff
@@ -0,0 +1,79 @@
+# Release checklist
+
+Copy this template to the release pull request description.
+
+## Update workflows
+
+- [ ] `.github/workflows/build.yml` and `.github/workflows/install-dependencies.sh`: Update the OpenFOAM and preCICE versions, if needed.
+- [ ] `build-custom.yml`: Add any new OpenFOAM versions and update the default preCICE version, if needed.
+
+## Prepare branches
+
+- [ ] Merge `master` to `develop`, if needed
+- [ ] Open a pull request from `develop` to `master`.
+  - We don't use release branches, unless there are changes we don't want to release, potentially when preparing a major release.
+- [ ] Run and fix all workflows
+
+## Test
+
+- [ ] Run and check flow-over-heated-plate OpenFOAM-OpenFOAM
+- [ ] Run and check quickstart
+- [ ] Run and check perpendicular-flap OpenFOAM-deal.II
+- [ ] Run and check elastic-tube-3d OpenFOAM-CalculiX
+- [ ] Run and check partitioned-pipe sonicLiquidFoam-sonicLiquidFoam
```

Comment:
How do I check cases without reference results?

Reply:
These are meant to be regressions checks. In the future, we will have the system tests comparing the interface VTK files. For now, most problems would appear just when running the simulation. If the simulations does not start, crashes, or the results look qualitatively different, then there is definitely a problem. If this is ok, I would try to compare some basic results to the previous release (depending on the case), especially if there are significant changes in the adapter.

I also think it is a good idea to always add a detailed comment with what I checked and what I observed, as well as mention what I did not check.